src/test/groovy/graphql/nadel/NadelTest.groovy

```diff
@@ -72,13 +72,13 @@ class NadelTest extends Specification {
                 id: ID 
                 title : String 
                 barId: ID
-                bar : String <= \$innerQueries.BarService.bar(id: \$source.barId)
+                bar : String <= \$innerQueries.BarService.bar(barId: \$source.barId)
             }
         }
         
         service BarService {
             type Query {
-                bar(id: ID): Bar
+                bar(barId: ID): Bar
```

Comment:
We don't really have a test, that barId is actually is used as Argument for `bar`: the mocks respond in a very generic way (may fault). Do you think we can make it more strict to actually test that the right argument is used?

Reply:
change test so that i really checks the argument